### ISSUE TITLE
hidding table names on query results and data tab

### ISSUE THREAD

**Issue by thiagonogueira** [dbeaver/dbeaver#2797]
I'd like to show just the column name on query results and when navigating over a object data tab. Couldn't find how to. Follow some examples that show the results with table names:

![image](https://user-images.githubusercontent.com/755318/35008972-69a82a04-fae5-11e7-9f16-1de10ee499f2.png)

![image](https://user-images.githubusercontent.com/755318/35009069-b6887860-fae5-11e7-872d-848745763c49.png)

**Comment by serge-rider** [comment]
I'm not quite understand. 
You need just column list and don't need data itself?

Try to click Panels->Metadata

**Comment by serge-rider** [comment]
I ca't reproduce such behavior with any available database/driver. DBeaver doesn't show table names as a part of column names. This should be some specific behavior of the driver.
What database do you use?

(Sorry for so long reply)

**Comment by serge-rider** [comment]
This is Hive-specific behavior. By some reason it returns column names in such weird form.

Potentially we could make some workaround but I'm not sure we should. Maybe we should file a bug report to Hive dev..

**Comment by cmrichard** [comment]
Run this set command in your dbeaver SQL Editor and you should be good to go. 
`set hive.resultset.use.unique.column.names=false;`

**Comment by asmoth360** [comment]
It works when I set the variable with : 
`set hive.resultset.use.unique.column.names=false;`

But it resets to true everytime I reopen DBeaver. How can I do to keep it that way ? I didn't find in the configuration,

**Comment by cmrichard** [comment]
Try using Bootstrap Queries like this: 
1.) Enter the Connection Wizard to re-create your Hive Settings (hit "next" three times)
2.) On the screen "Finish General Connection Settings", 
    --> Click the url hyperlink to enter: "Connection initialization settings"
    --> Hit the "Configure" button under "Bootstrap Queries:" and add your SQL queries.
...
Note: I didn't test this (we use a labor-intensive, bastardized config that isn't wizard-friendly for hive), 
but its something to chew-on and may work out of the box for you. Try the "Shell Commands" url if bootstrap goes nowhere. 

I actually stopped using the SET method workaround I posted.
In the record results-window, the Filter button stops listing the distinct entries per field, meaning you'd have to eyeball thousands of records of categorical data to spot the one stray/undefined entry that shouldn't exist in that column and was inserted incorrectly. Definitely a big dealbreaker for me. 

My new "Learn a better habit" fix is: Alias tables that use asterisk queries to something short.

Query:      Select * FROM  ExtraLongNamespaceTableHere  t  ;   
Returns:    t.foo  , t.bar ,  t.bat ,  t.baz ...
